frontend/src/components/common/Game/AdminMenu.vue

```diff
@@ -86,6 +87,31 @@ async function switchFromFavourites() {
       emitter?.emit("showLoadingDialog", { loading: false, scrim: false });
     });
 }
+
+async function resetLastPlayed() {
+  await romApi
+    .updateUserRomProps({
+      romId: props.rom.id,
+      data: { last_played: null },
+    })
+    .then(() => {
+      emitter?.emit("snackbarShow", {
+        msg: `${props.rom.name} removed from Continue Playing`,
```

Comment:
Do we currently supporting i18n here?

Reply:
Yes, but we already have a ton of these error messages that aren't translated. At some point we'll do a second pass and correct these, but since they're transient IMO they're lower priority.